src/python/grapl_analyzerlib/grapl_analyzerlib/schema.py

```diff
@@ -1,10 +1,20 @@
 import abc
+import logging
+import os
+import sys
 import types
+from typing import cast, Callable, Type, TypeVar, Any, Dict, Tuple, Union
 
-from copy import deepcopy
+from grapl_analyzerlib.grapl_client import GraphClient
 
-from typing import cast, Type, TypeVar, Any, Dict, Tuple, Union
+IS_LOCAL = bool(os.environ.get("IS_LOCAL", False))
```

Comment:
a) we should consider bringing super important variables like this to somewhere unified
b) let's make it as immutable as we can: `IS_LOCAL: typing_extensions.Final[bool] = ...`